### PR TITLE
Remove properties file, hardcode instead.

### PR DESCRIPTION
**Pull request by brian-brazil**
The port and binding property still exist, and
can be set on the command line with -D.

### CODE REVIEW

File: src/com/carolinarollergirls/scoreboard/Main.java [hunk 1]
```diff
@@ -52,11 +52,6 @@ private void parseArgv(String[] argv) {
             else if(arg.equals("--nogui") || arg.equals("-G")) {
                 gui = false;
             }
-
-            else if(arg.startsWith("--port=") || arg.startsWith("-p=")) {
-                String port = arg.split("=")[1];
-                ScoreBoardManager.setPropertyOverride("com.carolinarollergirls.scoreboard.jetty.JettyServletScoreBoardController.port", port);
```
Comment (frank-weinberg) [inline src/com/carolinarollergirls/scoreboard/Main.java:58]: Why remove this? `-p=` is a lot easier to remember and type than `-Dcom.carolinarollergirls.scoreboard.jetty.JettyServletScoreBoardController.port=`. I'd rather think about adding something like `-l` and `--localhost` for that property.

Comment (brian-brazil) [reply src/com/carolinarollergirls/scoreboard/Main.java:58]: It's a fairly advanced feature that few users need (or are even aware exists), plus two ways to configure things is asking for confusion about which takes precedence.

Comment (frank-weinberg) [reply src/com/carolinarollergirls/scoreboard/Main.java:58]: "Few users need it, so we can make it significantly more obscure" is not a convincing argument. And there's no need to even document the `-D` way - we can just treat it as something that happens to have that effect but is not an official feature at which point everybody who is able to figure out that it exists should also be able to figure out that `-p`/`-l` takes precedence (or that they should just leave it alone).

Comment (brian-brazil) [reply src/com/carolinarollergirls/scoreboard/Main.java:58]: Updated to take a different approach.

File: src/com/carolinarollergirls/scoreboard/ScoreBoardManager.java [hunk 2]
```diff
@@ -118,41 +111,9 @@ private static void loadVersion() {
         printMessage("Carolina Rollergirls Scoreboard version "+getVersion());
     }
 
-    private static void loadProperties() {
```
Comment (frank-weinberg) [inline src/com/carolinarollergirls/scoreboard/ScoreBoardManager.java:119]: This leaves `FileInputStream`, `FileNotFoundException`, `HashMap`, `Map`, and `ConcurrentHashMap` unused.

File: src/com/carolinarollergirls/scoreboard/jetty/JettyServletScoreBoardController.java [hunk 3]
```diff
@@ -67,27 +67,12 @@ public JettyServletScoreBoardController(ScoreBoard sb, JSONStateManager jsm) {
...
-            ScoreBoardManager.printMessage("ScoreBoard configured to listen ONLY on localhost interface.");
-            SocketConnector sC = new SocketConnector();
-            sC.setHost("localhost");
-            sC.setPort(port);
-            server = new Server();
-            server.addConnector(sC);
-        } else {
-            server = new Server(port);
-        }
+    protected void init(String host, int port) {
```
Comment (frank-weinberg) [inline src/com/carolinarollergirls/scoreboard/jetty/JettyServletScoreBoardController.java:69]: This shadows a class attribute (that appears to be unused)

File: src/com/carolinarollergirls/scoreboard/jetty/JettyServletScoreBoardController.java [hunk 4]
```diff
@@ -128,27 +113,11 @@ protected void init() {
         HttpServlet ljs = new LoadJsonScoreBoard(scoreBoard);
         c.addServlet(new ServletHolder(ljs), "/LoadJSON/*");
 
-        Enumeration<?> keys = ScoreBoardManager.getProperties().propertyNames();
```
Comment (frank-weinberg) [inline src/com/carolinarollergirls/scoreboard/jetty/JettyServletScoreBoardController.java:131]: `Enumeration` is now unused

File: src/com/carolinarollergirls/scoreboard/jetty/MediaServlet.java [hunk 5]
```diff
@@ -32,18 +33,18 @@
...
+    public MediaServlet(ScoreBoard sb, String dir) {
+        scoreBoard = sb;
+        htmlDirName = dir;
     }
 
-    @Override
-    public String getPath() { return "/Media"; }
-
-    @Override
     protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException,IOException {
```
Comment (frank-weinberg) [inline src/com/carolinarollergirls/scoreboard/jetty/MediaServlet.java:45]: This method is still `@Override`

**Comment by brian-brazil** [comment]
All fixed.

**Pull request merged**